docs/_src/tutorials/tutorials/1.md

```diff
@@ -237,6 +237,36 @@ prediction = pipe.run(
 
 
 ```python
+# Now you can either print the object directly...
+from pprint import pprint
+
+pprint(prediction)
+
+# Sample output:    
+# {
+#     'answers': [ <Answer: answer='Eddard', type='extractive', score=0.9919578731060028, ... >,
```

Comment:
Let's show a few more fields here per answer (I think offsets and context are important) and rather fewer answer objects in the list (e.g. just 2).